src/uct/ib/efa/srd/srd_iface.c

```diff
@@ -922,23 +933,8 @@ uct_srd_iface_query(uct_iface_h tl_iface, uct_iface_attr_t *iface_attr)
     iface_attr->cap.get.min_zcopy =
             iface->super.config.max_inl_cqe[UCT_IB_DIR_TX] + 1;
 
-    if (iface_attr->cap.am.max_bcopy > 0) {
-        iface_attr->cap.flags |= UCT_IFACE_FLAG_AM_BCOPY;
-    }
-
-    if (iface_attr->cap.am.max_zcopy > 0) {
-        iface_attr->cap.flags |= UCT_IFACE_FLAG_AM_ZCOPY;
-    }
-
-    if (iface_attr->cap.am.max_short > 0) {
-        iface_attr->cap.flags |= UCT_IFACE_FLAG_AM_SHORT;
-    }
-
-    if (iface_attr->cap.get.max_bcopy > 0) {
+    if (iface->config.max_get_zcopy > 0) {
```

Comment:
can we set
iface_attr->cap.flags |= UCT_IFACE_FLAG_GET_BCOPY|UCT_IFACE_FLAG_GET_ZCOPY